docs/ops/condition/NonZero_3.md

```diff
@@ -4,34 +4,37 @@
 
 **Category**: Condition operation
 
-**Short description**: *NonZero* returns the indices of the non-zero elements of the input tensor.
-
-**Detailed description**: *NonZero* returns the indices of the non-zero elements of the input tensor (in row-major order - by dimension).
-The output tensor has shape `[rank(input), num_non_zero]`. For example, for the tensor `[[1, 0], [1, 1]]` the output will be `[[0, 1, 1], [0, 0, 1]]`.
+**Short description**: *NonZero* returns the indices of the non-zero elements of the input tensor. 
 
 **Attributes**
 
 * *output_type*
-
   * **Description**: the output tensor type
-  * **Range of values**: "i64" or "i32"
+  * **Range of values**:
+    * *i64*
+    * *i32*
   * **Type**: string
   * **Default value**: "i64"
   * **Required**: *No*
 
 **Inputs**:
 
-*   **1**: `data` tensor of arbitrary rank of type *T*. Required.
+*   **1**: `data` tensor of arbitrary rank of type *T*. **Required**.
 
 **Outputs**:
 
-*   **1**: tensor with indices of non-zero elements of shape `[rank(data), num_non_zero]` of type *T_IND*.
+*   **1**: tensor with indices of non-zero elements of shape `[rank(data), num_non_zero]` of type *T_OUT*.
 
 **Types**
 
 * *T*: any numeric type.
 
-* *T_IND*: `int64` or `int32`.
+* *T_OUT*: Depending on *output_type* attribute can be `int64` or `int32`.
+
+**Detailed description**: *NonZero* returns the indices of the non-zero elements of the input tensor (in row-major order - by dimension).
```

Comment:
I'm not sure why you moved it here, convention is to have it after short description.